tests/integration/testlib/ovslib.py

```diff
@@ -135,16 +135,18 @@ def _set_ifaces_state(ifaces, state):
     return ifaces
 
 
-def get_proxy_port_name_of_ovs_interface(iface_name):
-    output = cmdlib.exec_cmd("ovs-vsctl show".split(" "), check=True)[1]
-    port_name = None
-    port_line_re = re.compile(r'^\s+Port\s"(.+)"$')
-    iface_line_re = re.compile(rf'^\s+Interface\s"{iface_name}"$')
-    for line in output.split("\n"):
-        match = port_line_re.match(line)
+def get_proxy_port_profile_name_of_ovs_interface(iface_name):
+    proxy_port_iface_name = cmdlib.exec_cmd(
```

Comment:
Q: If the master is set, can it be that there is no profile for the master?

I'm not exactly sure, but what happens if the interface profile is not active? Same for the proxy/port profile.

Reply:
When mater been set while master does not exists in any profile, the activation will fail.

The command line `nmcli` will show both activated/deactivated profiles. If that's the second questions for.

Reply:
Added `--active` to show the only activated.